index.bs

```diff
@@ -2089,6 +2091,48 @@ To <dfn>await a navigation</dfn> given |context|, |request|, |wait condition|, a
 
 </div>
 
+#### The browsingContext.Locator Type #### {#type-browsingContext-Locator}
+
+[=remote end definition=] and [=local end definition=]
+
+<pre class="cddl remote-cddl local-cddl">
+browsingContext.Locator = (
+   browsingContext.CssLocator /
+   browsingContext.FrameLocator /
+   browsingContext.ScriptLocator /
+   browsingContext.TextLocator /
+   browsingContext.XPathLocator
+)
+
+browsingContext.CssLocator = {
+   type: "css",
+   value: text
+}
+
+browsingContext.FrameLocator = {
+   type: "frame"
```

Comment:
Is the `context nodes` parameter missing here?

Reply:
It is not. This locator is only chainable; it can't be used on its own. The intent is to combine it with whatever locator returns the frame/iframe element. All this does is switch the "active" context. As a side effect, it also enables the classic "switch to frame" command.